Classes/Hooks/DataHandler.php

```diff
@@ -38,20 +39,11 @@ class DataHandler implements LoggerAwareInterface
     use LoggerAwareTrait;
 
     /**
+     * @Inject
```

Comment:
I wonder about the @Inject annotation. Does this really work at this place?

Reply:
No it doesn't. :-/

Reply:
:-)